### ISSUE TITLE
docker image does not include 'pymssql'

### ISSUE THREAD

**Issue by ishioni** [apache/airflow#8712]
**Apache Airflow version**: 1.10.10

**Kubernetes version (if you are using kubernetes)** (use `kubectl version`): 1.15.11

**Environment**: dockerhub image apache/airflow:1.10.10

**What happened**:
When importing from airflow.hooks.mssql_hook import MsSqlHook, airflow gives an error No module named 'pymssql'

**What you expected to happen**:
It should work :)

The docker image does not include pymssql from pip

**Comment by potiuk** [comment]
Thanks! Will take a look!

**Comment by alexbegg** [comment]
pymssql is installed with the airflow extra "mssql". It is not pre-installed with the docker image apache/airflow:1.10.10. I am currently installing it as a separate pip install of `apache-airflow[mssql]==1.10.10` in my helm chart and I have no issue using the MsSqlHook.

**Comment by potiuk** [comment]
Yep. We want to minimise the prod image size to get the most frequently used packages (and core ones). We've added a new feature today to the prod image - you can add additional extras or python  deps  and b build your own version of the image. https://github.com/apache/airflow/blob/master/IMAGES.rst#production-images

Breeze supports building they production image and it wil even help you to add the extras in. For example:

```
./breeze build-image --production-image --additional-extras mssql
```

So I consider this solved with this feature via #8866
